### PR TITLE
docs(api): Remove use of basic Pipette constructor from docs

### PR DESCRIPTION
**Pull request by btmorr**
## overview

Remove straggling use of plain `Pipette` constructor in docs, and some other OT1 holdovers.

## changelog

- docs(api): Remove use of basic Pipette constructor from docs

### CODE REVIEW

File: api/docs/source/index.rst [hunk 1]
```diff
@@ -28,12 +28,12 @@ The design goal of the Opentrons API is to make code readable and easy to unders
 
 .. code-block:: none
 
-    Use the Opentrons API's containers and instruments
+    Use the Opentrons API's labware and instruments
 
-    Add a 96 well plate, and place it in slot 'B1' of the robot deck
-    Add a 200uL tip rack, and place it in slot 'A1' of the robot deck
+    Add a 96 well plate, and place it in slot '2' of the robot deck
```
Comment (mcous) [inline api/docs/source/index.rst]: Same slot change needs to be made in the above text block

File: api/docs/source/index.rst [hunk 2]
```diff
@@ -45,11 +45,11 @@ If we were to rewrite this with the Opentrons API, it would look like the follow
     from opentrons import containers, instruments
```
Comment (mcous) [inline api/docs/source/index.rst]: `from opentrons import labware, instruments`

File: api/docs/source/index.rst [hunk 3]
```diff
@@ -62,48 +62,48 @@ How it's Organized
...
 
-From the example above, the "containers" section looked like:
+From the example above, the "labware" section looked like:
 
 .. code-block:: python
 
-    plate = containers.load('96-flat', 'B1')
-    tiprack = containers.load('tiprack-200ul', 'A1')
+    plate = labware.load('96-flat', '2')
+    tiprack = labware.load('tiprack-200ul', 'A1')
```
Comment (mcous) [inline api/docs/source/index.rst]: `A1` should be `1`

File: api/docs/source/atomic commands.rst [hunk 4]
```diff
@@ -150,7 +150,7 @@ Calls to ``pick_up_tip()`` will by default start at the attached tip rack's ``'A
 
 .. code-block:: python
 
-    pipette.start_at_tip(tip_rack_1['C3'])
+    pipette.start_at_tip(tip_rack_1.well('C3'))
```
Comment (mcous) [inline api/docs/source/atomic commands.rst]: `C3` should be an OT-2 slot

Comment (btmorr) [reply api/docs/source/atomic commands.rst]: This is a well, not a slot

File: api/docs/source/index.rst (hunk 1 above)
Comment (btmorr) [reply api/docs/source/index.rst]: Same as above

File: api/docs/source/pipettes.rst [hunk 5]
```diff
@@ -59,3 +59,13 @@ The values are in microliters/seconds, and have varying defaults depending on th
         aspirate_flow_rate=200,
         dispense_flow_rate=600)
 
+
+
+Old Pipette Constructor
+=======================
+
+The ``Pipette`` constructor that was used directly in OT-One protocols is not
```
Comment (mcous) [inline api/docs/source/pipettes.rst]: `s/not/now`

File: api/docs/source/pipettes.rst [hunk 6]
```diff
@@ -59,3 +59,13 @@ The values are in microliters/seconds, and have varying defaults depending on th
...
+an internal-only class. Its behavior is difficult to predict when not used
+through the public constructors mentioned above. ``Pipette`` constructor
+arguments are subject to change of their default values, behaviors, and
+parameters may be added or removed within a major version of the API.
```
Comment (mcous) [inline api/docs/source/pipettes.rst]: > ``Pipette`` constructor arguments are subject to change of their default values, behaviors, and parameters may be added or removed within a major version of the API.

...is accurate, but "within a major version" may be a little hard to parse for someone not well versed in semver. I wonder if we could simplify it at all? Maybe something along the lines of:

> The internal ``Pipette`` constructor is subject to change without warning or a major version bump.
